src/background/Wallet/Wallet.ts

```diff
@@ -1049,11 +1049,26 @@ export class Wallet {
     }
   }
 
+  private getOfflineSignerByAddress(address: string) {
```

Comment:
Had to do the same thing I did in #629 cuz it is not merged yet

Reply:
Actually I'll better try to re-use `SignMessageButton` (hence I don't need these changes in this PR)